packages/client/src/proxy.js

```diff
@@ -59,14 +59,22 @@ export function href(options) {
     (path || `${pathname || ''}${search || ''}${hash || ''}`);
 };
 
+// auto strip double quotes/spaces if any
+function stripQuotesAndSpaces(line) {
```

Comment:
some linting issues, but we will also need to do it for other env vars, so maybe move this function to util package